src/utils/progress.rs

```diff
@@ -42,6 +42,16 @@ impl ProgressBar {
     }
 
     pub fn finish_with_message(&self, msg: &str) {
+        self.inner.finish_with_message(msg.to_owned());
+        logging::set_progress_bar(None);
+    }
+
+    pub fn finish_with_duration(&self, op: &str) {
```

Comment:
you think we can include extra data other than the operation here so the message is more detailed?

Reply:
Sure we could. Anything you have in mind? I kept it concise so it reads nicely 😅

Reply:
Yeah I got nothing for now, but I guess we can always come back and iterate.